src/mSupplyMobileApp.js

```diff
@@ -68,23 +68,35 @@ class MSupplyMobileAppContainer extends React.Component {
     this.scheduler = new Scheduler();
     const isInitialised = this.synchroniser.isInitialised();
     this.scheduler.schedule(this.synchronise, SYNC_INTERVAL);
+    // this.scheduler.schedule(this.synchroniseSensor, SYNC_INTERVAL);
     this.scheduler.schedule(() => {
       const { currentUser } = this.state;
       if (currentUser !== null) {
         // Only reauthenticate if currently logged in.
         this.userAuthenticator.reauthenticate(this.onAuthentication);
       }
     }, AUTHENTICATION_INTERVAL);
+
+    const latestTemperatureSync = database.objects('SensorLog').max('timestamp');
     this.state = {
       confirmFinalise: false,
-      currentUser: null,
       isInitialised,
       isLoading: false,
+      currentUser: null,
       syncModalIsOpen: false,
+      temperatureSyncState: {
+        message: 'Temperature Sync Details',
+        lastSync: latestTemperatureSync,
+        isSyncing: false,
+        progress: 1,
+        total: 1,
+      },
     };
   }
 
-  componentDidMount = () => BackHandler.addEventListener('hardwareBackPress', this.handleBackEvent);
+  componentDidMount = async () => {
```

Comment:
ooh, is this new ? why async ?

Reply:
just an accident from devving from that method hehe